src/types/types.ts

```diff
@@ -1,16 +1,19 @@
-import {
+import type {
   IEncryptedMessage,
   DidUri,
   KiltAddress,
   DidResourceUri,
 } from '@kiltprotocol/types'
-import { HexString } from '@polkadot/util/types'
-import { SelfSignedProof, VerifiableCredential } from '@kiltprotocol/vc-export'
+import type { HexString } from '@polkadot/util/types'
+import type {
+  CredentialDigestProof,
+  SelfSignedProof,
+  VerifiableCredential,
+  constants,
```

Comment:
Is constants a type? They are imported as a type.

Reply:
they are used as a type too

Reply:
I thought that `import type` was added so that such imports would be dropped during transpilation. So, in my understanding, mixed imports are safer to have without the `type` modifier.

Reply:
no, the point is, we are not using the value, we are using the type. if we did use its value, ts would complain